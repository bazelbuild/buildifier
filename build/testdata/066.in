load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
load("@bazel_tools//tools/build_defs/pkg:rpm.bzl", "pkg_rpm")

pkg_tar(
    name = "test_tar",
    srcs = [],
)

pkg_rpm(
    name = "test_rpm",
    srcs = [],
)
