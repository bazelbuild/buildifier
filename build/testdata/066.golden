load("@rules_pkg//:pkg.bzl", "pkg_tar")
load("@rules_pkg//:rpm.bzl", "pkg_rpm")

pkg_tar(
    name = "test_tar",
    srcs = [],
)

pkg_rpm(
    name = "test_rpm",
    srcs = [],
)
